<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgHsvOption</class>
 <widget class="QWidget" name="WdgHsvOption">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>406</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>255</horstretch>
    <verstretch>255</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>385</width>
    <height>400</height>
   </size>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="hueTab">
      <attribute name="title">
       <string>Hue</string>
      </attribute>
      <widget class="KisCurveWidget" name="hueCurve" native="true">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>361</width>
         <height>251</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="modeHueCBox">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>320</y>
         <width>83</width>
         <height>27</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Grow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Shrink</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Random</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pressure</string>
        </property>
       </item>
      </widget>
      <widget class="KDoubleNumInput" name="inkAmountHue">
       <property name="geometry">
        <rect>
         <x>119</x>
         <y>290</y>
         <width>276</width>
         <height>28</height>
        </rect>
       </property>
       <property name="value">
        <double>100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>20.000000000000000</double>
       </property>
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="sliderEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="inkAmountLbl3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>290</y>
         <width>75</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Ink amount</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="saturationTab">
      <attribute name="title">
       <string>Saturation</string>
      </attribute>
      <widget class="KisCurveWidget" name="saturationCurve" native="true">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>361</width>
         <height>251</height>
        </rect>
       </property>
      </widget>
      <widget class="KDoubleNumInput" name="inkAmountSaturation">
       <property name="geometry">
        <rect>
         <x>119</x>
         <y>280</y>
         <width>276</width>
         <height>28</height>
        </rect>
       </property>
       <property name="value">
        <double>100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>20.000000000000000</double>
       </property>
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="sliderEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="inkAmountLbl2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>280</y>
         <width>75</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Ink amount</string>
       </property>
      </widget>
      <widget class="QComboBox" name="modeSaturationCBox">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>310</y>
         <width>83</width>
         <height>27</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Grow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Shrink</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Random</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pressure</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QWidget" name="valueTab">
      <attribute name="title">
       <string>Value</string>
      </attribute>
      <widget class="KisCurveWidget" name="valueCurve" native="true">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>361</width>
         <height>251</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>41</x>
         <y>271</y>
         <width>357</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="inkAmountLbl">
          <property name="text">
           <string>Ink amount</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="KDoubleNumInput" name="inkAmountValue">
          <property name="value">
           <double>100.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>20.000000000000000</double>
          </property>
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="sliderEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QComboBox" name="modeValueCBox">
          <item>
           <property name="text">
            <string>Grow</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Shrink</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Random</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Pressure</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>268</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KDoubleNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KisCurveWidget</class>
   <extends>QWidget</extends>
   <header>widgets/kis_curve_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
